### PR TITLE
refactor: Slightly reorganize Kubernetes how-tos

### PR DESCRIPTION
**Pull request by fghaas**
* Add `index.md` files in the `howto/kubernetes/magnum` and `howto/kubernetes/gardener` directories.
* Move generic information about Gardener and Magnum into those files.
* Clarify that Gardener is in closed beta, and add instructions on how to enroll in it.

### CODE REVIEW

File: docs/howto/kubernetes/index.md [hunk 1]
```diff
@@ -1,40 +1,9 @@
 # Kubernetes in Cleura
 
-<!-- Config interpolation for the company name didn't work for the 
-title here. -->
+In {{brand}} Cloud you can run Kubernetes in several
+ways.
```
Comment (colder-is-better) [inline docs/howto/kubernetes/index.md:5]: Since you don't _run_ Kubernetes, I would rephrase this sentence as follows: 

In {{brand}} Cloud there are several ways to deploy a Kubernetes cluster.

**Review by colder-is-better — COMMENTED** [review]
Looks good --- I have just one suggestion.

File: docs/howto/kubernetes/index.md (hunk 1 above)
Comment (fghaas) [reply docs/howto/kubernetes/index.md:5]: Fair point; I used not quite that but something similar. :)
